main.cpp

```diff
@@ -566,6 +567,7 @@ int main(int argc, char *argv[])
 	cbs.add("--cpp", [&args](CLIParser &) { args.cpp = true; });
 	cbs.add("--cpp-interface-name", [&args](CLIParser &parser) { args.cpp_interface_name = parser.next_string(); });
 	cbs.add("--metal", [&args](CLIParser &) { args.metal = true; });
+	cbs.add("--msl-no-pack-ubos", [&args](CLIParser &) { args.msl_pack_ubos = false; });
```

Comment:
Not quite the same imo, as it's Metal vs Metal Shading Language, but HLSL would be --high-level-shading-language, but anyways, seems sensible to use msl throughout the code base if we're going to choose between metal and msl.

Only thing that needs to remain is --metal in the CLI for compatibility (but help should say --msl, and --msl should be supported as well).
If using --metal option, an stderr message could say it's deprecated in favor of --msl.